sympy/utilities/tests/test_pickling.py

```diff
@@ -166,8 +166,10 @@ def test_core_undefinedfunctions_fail():
 
 
 def test_core_interval():
-    for c in (Interval, Interval(0, 2)):
-        check(c)
+    # Set.is_EmptySet is deprecated since 1.5
+    with ignore_warnings(SymPyDeprecationWarning):
+        for c in (Interval, Interval(0, 2)):
+            check(c)
```

Comment:
Not sure what this does but maybe the is_EmptySet should be fixed at source rather than ignoring the warnings here...

Reply:
What do you mean by "the is_EmptySet should be fixed at source"?

The `check` function ("Check that pickling and copying round-trips.") is rather technical (different pickling protocol versions etc.), so I didn't want to touch it and adding an ignore_warnings in there could potentially hide other bugs. As for is_EmptySet itself I could only remove the deprecation?

Reply:
Presumably somewhere inside `check` the `is_EmptySet` attribute is being used. If this PR is deprecating `is_EmptySet` then we should make sure it's not used anywhere in SymPy.

Reply:
I think that you can just add `is_EmptySet` to `excluded_attrs` at the top of the file.

Reply:
The `excluded_attrs` list should have comments explaining the reasons e.g.:
```
excluded_attrs = set([
    '_assumptions',  # This is a local cache that isn't automatically filled on creation
    '_mhash',   # Cached after __hash__ is called but set to None after creation
    'message',   # This is an exception attribute that is present but deprecated in Py2 (can be removed when Py2 support is dropped
    'is_EmptySet',  # Deprecated from SymPy 1.5. This can be removed when is_EmptySet is removed.
])
```

Reply:
Thanks, I think those comments are definitely valuable.